src/typescript.ts

```diff
@@ -15,6 +15,17 @@ function nameIsReservedKeyword(name: string): boolean {
   return reservedKeywords.indexOf(name) !== -1;
 }
 
+/**
+ * Will determine whether the tableName should be prefixed with the schemaName
+ */
+export function getEnhancedTableName(
+  tableName: string,
+  schema: string,
+  prefixWithSchemaNames: boolean,
+) {
+  return `${prefixWithSchemaNames ? `${schema}_` : ''}${tableName}`;
```

Comment:
nit: avoid nesting template strings, I think this is clearer as:

```ts
return (prefixWithSchemaNames ? `${schema}_` : '') + tableName;
```

Reply:
Nice! I like that solution